build/buildall

```diff
@@ -39,7 +39,7 @@ SPARK_SHIM_VERSIONS="
 BUILD_PARALLELISM=1
 
 time (
-echo -n "$SPARK_SHIM_VERSIONS" | xargs -I% -P $BUILD_PARALLELISM -n 1 \
+echo -n "$SPARK_SHIM_VERSIONS" | xargs -t -I% -P $BUILD_PARALLELISM -n 1 \
```

Comment:
nit, debug?

Reply:
From just looking at the build log it is very hard to tell what version of Spark is being built against. This gives a very clear clue.

Reply:
ack, sounds good